recipes/hdmf/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hdmf" %}
+{% set version = "0.post0.dev1081" %}
+{% set sha256 = "8b0264a51ef1aea5fb22d450a7fa6cbc74bac97421e9667d1b7caee0ab74ce57" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - h5py
+    - numpy
+    - scipy
+    - ruamel.yaml
+    - python-dateutil
+    - six
+    - requests
+
+test:
+  imports:
+    - hdmf
+
+about:
+  home: https://github.com/hdmf-dev/hdmf
+  license: BSD-3-Clause
+  license_file: license.txt
+  summary: 'A Python API for working with hierarchical object data'
+  description: |
+    The Hierarchical Data Modeling Framework, or HDMF is a Python package
+    for working with hierarchical data. It provides APIs for specifying data
+    models, reading and writing data to different storage backends, and
+    representing data with Python object.
+  doc_url: http://hdmf.readthedocs.io/en/latest/
+
+extra:
+  recipe-maintainers:
+    - ajtritt
+    - oruebel
```

Comment:
@oruebel - are you OK with being a maintainer here?

Reply:
@scopatz yes, that is fine.